components/todo-list.js

```diff
@@ -1,19 +1,19 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState } from 'react';
 
 import TodoItem from "./todo-item";
 
-export default function TodoList() {
+import styled from "styled-components";
+
+export default function TodoList({todos, todoRemove}) {
+
```

Comment:
```suggestion
export default function TodoList({todos, onDelete}) {

```
props 이름을 onDelete로 하는 게 좋습니다. 그 이유는 ```pages/index.js```의 리뷰를 확인해주세요!